saleor/graphql/core/utils/__init__.py

```diff
@@ -33,15 +38,47 @@ def str_to_enum(name):
     return name.replace(" ", "_").replace("-", "_").upper()
 
 
-def validate_image_file(file, field_name):
+def validate_image_file(file, field_name, error_class):
```

Comment:
Later in this func we call either `error_class.REQUIRED` or `error_class.INVALID`. Using this func anywhere else I would depend on signature and pass any error class of my wish (not necessary having these attributes) which would raise `AttributeError`. I think we should at least add typing hints here.

Reply:
Typing hints will not help here as even if I specified that it should be enum it will not imply that this enum will have `REQUIRED` and `INVALID` fields.